README.md

```diff
@@ -52,3 +52,18 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Running unit and integration tests
+
+To run the tests make sure pytest>=7.4.2  is installed 
```

Comment:
In terms of versioning, we should ensure this is true by making sure it's enforced through poetry; so anyone who's run `poetry install` should have a valid version